penaltymodel_lp/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+FACTORY_ENTRYPOINT = 'penaltymodel_factory'
+
+install_requires = ['dimod>=0.6.0,<0.8.0',
+                    'penaltymodel>=0.15.0,<0.16.0',
+                    'scipy>=0.15.0,<2.0.0',
+                    'numpy>=0.0.0,<1.16.0'
+                    ]
+
+packages = ['penaltymodel',
+            'penaltymodel.lp',
+            ]
+
+setup(
+    name="penaltymodel-lp",
+    install_requires=install_requires,
+    entry_points={
```

Comment:
Need to add the packages

```
setup(
    name="penaltymodel-lp",
    install_requires=install_requires,
    packages=packages,
    entry_points={
        FACTORY_ENTRYPOINT: ['lp = penaltymodel.lp:get_penalty_model']
    }
)
```

In general this needs to have additional parameters added. I would use https://github.com/dwavesystems/penaltymodel/blob/1b67fbe6f323373e20cebaad607a6a132b005620/penaltymodel_mip/setup.py#L1 as a template

Reply:
except `python_requires` should be
```
python_requires = '>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*'
```

Reply:
Okay. I'll push the change with the `packages` first to make sure it fixes the CI errors. If that passes,  I'll switch over and follow mip's `setup.py`.